cep-XXXX.md

```diff
@@ -0,0 +1,424 @@
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]/<OCI-compatible subdir>/patch_instructions.json:latest
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]/channeldata.json:latest
+```
+
+The various parts of the URL are defined below.
+
+### Channel Base URL and Subdirs
+
+The base URL for an OCI conda channel MUST be formatted as follows:
+
+```text
+oci://<authority>/<OCI-compatible channel path>[/label/<OCI-compatible label>]
+```
+
+where `<authority>` is any valid [RFC 3986][rfc3986] authority which is also supported by the OCI `v1.*` specification.
+This string will typically be `<hostname>[:<port>]` (e.g., `ghcr.io` with no port specified). OCI conda channel URLs MUST
+use `oci` as the scheme. OCI-compatible channel path and labels are defined below. If the `<OCI-encoded label>` is `main`,
+the entire optional label component MUST be omitted. Otherwise, the label component MUST be present.
+
+The base URL for an OCI conda channel is the canonical, unique identifier for the channel itself and SHOULD be used in
+conda ecosystem tools to refer to the channel.
+
+Conda `<OCI-compatible subdir>`s are appended to this base URL as follows:
+
+```text
+<OCI conda channel base URL>/<OCI-compatible subdir>
+```
+
+where `<OCI-compatible subdir>` is defined below.
+
+The following regexes MUST be satisfied by the `<OCI-compatible channel path>`, `<OCI-compatible subdir>`, and `<OCI-compatible label>`:
+
+- `<OCI-compatible channel path>`: `^[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*(\/[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*)*$` (same as the regex for an OCI repository `<name>`)
+- `<OCI-compatible label>`: `^[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*(\/[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*)*$` (same as the regex for an OCI repository `<name>`)
+- `<OCI-compatible subdir>`: `^[a-z0-9]+(-[a-z0-9]+)*$`
+
+### Subdir Repository Namespace Designations
+
+The namespace after the combination `<OCI conda channel base URL>/<OCI-compatible subdir>/` has the following designations
+(where the `*` wildcard indicates any valid OCI repository `<name>` component or part of a component as allowed by
+the OCI repository `<name>` regex):
+
+- `<OCI conda channel base URL>/<OCI-compatible subdir>/r*`: reserved for repodata JSON artifacts, run exports JSON artifacts, and
+  future CEP-defined extensions for sharded repodata (defined in [CEP 16][cep16]).
+- `<OCI conda channel base URL>/<OCI-compatible subdir>/c*`: reserved for OCI-encoded conda package names (The character `c`
+  is prepended to the conda package name to indicate that it is an OCI-encoded conda package name as described below.)
+- `<OCI conda channel base URL>/<OCI-compatible subdir>/h*`: reserved for hashed OCI-encoded conda package names
+- `<OCI conda channel base URL>/<OCI-compatible subdir>/p*`: reserved for repodata patch instructions
+
+These designations are used to prevent namespace collisions between the various components of an OCI conda channel.
```

Comment:
Should we add a clause that prevents any other initial letters from being used altogether? Something like:

```suggestion
These designations are used to prevent namespace collisions between the various components of an OCI conda channel. Any other string prefixes MUST NOT be used.
```